Assets/Scripts/EnemyHealth.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyHealth : MonoBehaviour {
```

Comment:
I'd add a summary tag before the class declaration itself to explain what this is for. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/summary